src/api/firebase.js

```diff
@@ -173,19 +184,22 @@ export async function shareList(listPath, currentUserId, recipientEmail) {
 		sharedLists: arrayUnion(listDocumentRef),
 	});
 
-	// check that shared list appears in the invited user's lists
+	// Check that shared list appears in the invited user's lists
 	const userDocAfterUpdate = await getDoc(userDocumentRef);
 	if (userDocAfterUpdate.exists()) {
 		const userDataAfterUpdate = userDocAfterUpdate.data();
 		if (
 			userDataAfterUpdate.sharedLists &&
 			userDataAfterUpdate.sharedLists.some((ref) => ref.path === listPath)
 		) {
-			return 'List successfully shared';
+			return { status: 200, message: 'OK: List successfully shared' };
```

Comment:
Small thought here - while 'OK' corresponds with this status code, might feel weird to a user? Maybe this response could be prefaced with something else (or not prefaced): Or 'Success: List shared!'?

Reply:
Yea I noticed that too when I shared a list. Will make that edit before merge.